### PR TITLE
Feature/arboretum

### PR DESCRIPTION
**Pull request by madhuqmar**

### CODE REVIEW

**Comment by csmcallister** [comment]
Thanks for opening this fresh PR @madhuqmar. I should get a chance to review tonight or tomorrow. In the meantime, could you address the lint errors:

```
events/us_arboretum.py:49:80: E501 line too long (89 > 79 characters)
events/us_arboretum.py:88:1: C901 'get_event' is too complex (14)
events/us_arboretum.py:98:80: E501 line too long (80 > 79 characters)
events/us_arboretum.py:168:1: C901 'get_more_info' is too complex (13
```

**Comment by madhuqmar** [comment]
Working on them as we speak! Should have more time to focus on it this evening.

**Comment by madhuqmar** [comment]
Hi Scott, my updated PR has the linted script for US National Arboretum. I think the script for ANS has an issue to fix and that's why my PR didn't pass the checks fully. But do let me know if I'm wrong. I can look into the ANS script to see what needs to be fixed. Thanks!

File: events/us_arboretum.py [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+            'Event Currency Symbol': "$",
+            'Event Category': " "})
+        return event_data
+    else:
+        pass
+
+
+def main():
+    try:
+        events = get_events()
```
Comment (csmcallister) [inline events/us_arboretum.py:307]: @madhuqmar Looks like `get_events` is missing its argument here so I'm unable to run/test

File: tests/us_arboretum_test.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from os import path
+import sys
+import unittest
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from events.us_arboretum import main
+from .utils import schema_test_required, schema_test_all, schema_test_types
+
+class ANSTestCase(unittest.TestCase):
```
Comment (csmcallister) [inline tests/us_arboretum_test.py:8]: Please rename test case to something like `UsArborteumTestCase` and update doc string.

**Comment by csmcallister** [comment]
@madhuqmar I fixed the bugs making the tests fail. Feel free to rebase on the upstream master if you'd like. Not required though.

**Comment by csmcallister** [comment]
closing due to inactivity, although feel free to reopen!

**Pull request closed without merging**